### PR TITLE
add missing modules to docs index in sidebar

### PR DESCRIPTION
**Pull request by jwmueller**
not sure this is the best way to add these as drop-downs. Ideally in our sidebar, they should look something like these modules from the cleanlab repo:

Screen Shot 2023-04-07 at 1 54 36 AM

Preview:
https://cleanvision.readthedocs.io/en/jwmueller-docsmodules/

### CODE REVIEW

File: docs/source/cleanvision/issue_managers/index.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+==============
+
+.. automodule:: cleanvision.issue_managers
+   :autosummary:
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+
+.. toctree::
```
Comment (jwmueller) [inline docs/source/cleanvision/issue_managers/index.rst:11]: Why this list of issue types specifically?

File: docs/source/conf.py [hunk 2]
```diff
@@ -33,8 +33,11 @@
 ]
 
 templates_path = ["_templates"]
+
 exclude_patterns = ["cleanvision/issue_managers/*", "cleanvision/utils/*"]
 
+exclude_patterns = []
```
Comment (jwmueller) [inline docs/source/conf.py:39]: What does this do?

**Review by jwmueller — COMMENTED** [review]
Had some questions

File: docs/source/cleanvision/issue_managers/index.rst (hunk 1 above)
Comment (sanjanag) [reply docs/source/cleanvision/issue_managers/index.rst:11]: I didn't understand your question

Comment (jwmueller) [reply docs/source/cleanvision/issue_managers/index.rst:11]: nevermind sorry I misread something!

**Review by jwmueller — COMMENTED** [review]
LGTM go ahead and merge once you've fixed the accidental addition and re-built the docs to verify it all looks good with these changes

**Pull request merged**